internal/storage/fake_storage_util.go

```diff
@@ -45,6 +72,12 @@ func CreateFakeStorageServer(objects []fakestorage.Object) (*fakestorage.Server,
 	})
 }
 
+func CreateFakeStorageServerWithTestObject() (*fakestorage.Server, error) {
```

Comment:
now that you have updated bucket_handle_test and storage_handle_test, 
1. do you still need CreateFakeStorageServer method
2. Does CreateFakeStorageServerWithTestObject need to be public?
3. Same for GetTestFakeStorageObject, should it be public?

Reply:
fixed